### DISCUSSION TITLE
Feature: Custom Kick Message

### DISCUSSION THREAD

**Discussion by Broxzier** [OpenRCT2/OpenRCT2#15682] [Feature requests]
Taken from https://openrct2.org/forums/topic/1115-kick-notes/

> I am tired when I get kicked from a server because I did something wrong, yet I don't know what I did wrong.
> I think it would be cool if the owner of the server had the option to write a note on the kick notification. Here is an example:
> Normal: Kicked: Get out of server!
> Custom: Kicked: Deleting rides!

**Comment by Superjustinbros** [comment]
I'd love this. Other multiplayer games already have such a feature so why not?

**Comment by IntelOrca** [comment]
Already suggested in #2897 but not as a separate issue.

**Comment by Gymnasiast** [comment]
I prefer to keep this open then. Keeping track of multiple things in one issue is tedious. I'll just rip this out of #2897.

**Comment by Nubbie** [comment]
Would be useful **but** a bit worried how it might slow down kicks for spammers who are using the cluster tool, really hope the permissions for the cluster tool will be released first a.k.a #3241 

Example how I presume it will look like:
![example](https://cloud.githubusercontent.com/assets/17282384/14313854/507f24ac-fbf4-11e5-8d0d-fbfba03e5cfc.png)
